include/swift/Reflection/Records.h

```diff
@@ -19,6 +19,9 @@
 
 #include "swift/Basic/RelativePointer.h"
 
+const uint16_t SWIFT_REFLECTION_METADATA_VERSION_MAJOR = 1;
```

Comment:
Perhaps we should just have a single version number. Do we expect to handle major number mismatches differently from minor?